.prettierrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "trailingComma": "es5",
+    "tabWidth": 2,
+    "semi": true,
+    "singleQuote": true,
+    "editor.formatOnSave": true,
+    "bracketSpacing": true,
+    "printWidth": 200
+  }
```

Comment:
@mr-pavlenko Do we need "arrow function brackets" set ro null here? 
or is printWidth - 200, how about 140 ? same we have in tslint file
what are the best practices here ?

Reply:
also do we need parser: 'typescript' here ?

Reply:
@JimHolman1199 Config has been changed with this commit 67a7bd2ba6989661ede73ec248c459e4646ec364